packages/docs/src/stories/components/expandable.stories.ts

```diff
@@ -9,7 +9,11 @@ const { generateTemplate } = storybookTemplate('sd-expandable');
 /**
  * Used to display a brief summary that expands to reveal additional content.
  *
- *  **Related templates**:
+ * **Accessibility Hints:**
```

Comment:
```suggestion
 * **Accessibility Information:**
```
Keeping it consistent with Accessibility Information in #1670